components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.h

```diff
@@ -24,6 +24,21 @@
  Delegate for MDCBottomDrawerContainerViewController.
  */
 @protocol MDCBottomDrawerContainerViewControllerDelegate <NSObject>
+
+/**
+This method is called when the bottom drawer updates its value for scrollViewIsScrolledToBottom.
+
+@param containerViewController the container view controller of the bottom drawer.
+@param scrollViewIsScrolledToEndOfContent whether or not the scroll view is scrolled to the end of
+the content.
+*/
+- (void)
+    bottomDrawerContainerViewControllerDidUpdateValueForScrollViewIsScrolledToEndOfContent:
```

Comment:
would we be able to shorten the naming here? it's a bit harder to comprehend. maybe `bottomDrawerContainerViewControllerDidReachEndOfContent`?